app/controllers/appointments_controller.rb

```diff
@@ -0,0 +1,46 @@
+class AppointmentsController < ApplicationController
+  
+  before_action :set_users
+    
+  def new
+      @appointment = Appointment.new
+      @appointment.key = SecureRandom.hex(4).upcase
```

Comment:
are we adding another "key" in schema for appointment?

Reply:
Migration file and updated schema are in pull request